templates/repo/view_file.tmpl

```diff
@@ -9,6 +9,22 @@
 			<div class="text left gt-whitespace-pre">{{.FileWarning}}</div>
 		</div>
 	{{end}}
+
+	{{if not .ReadmeInList}}
+		<div id="repo-file-commit-box" class="ui top attached header list-header">
```

Comment:
```suggestion
		<div id="repo-file-commit-box" class="ui top attached header list-header gt-mb-4">
```
I agree that this whole UI should be adapted in another PR, but please at least add the margin for now.